packages/create-svelte/shared/+typescript/tsconfig.json

```
@@ -10,4 +10,8 @@
 		"sourceMap": true,
 		"strict": true
 	}
+	// Path aliases are handled by https://kit.svelte.dev/docs/configuration#alias and https://kit.svelte.dev/docs/configuration#files
```

I don't think we need the reference to https://kit.svelte.dev/docs/configuration#files. The only thing there is `$lib` and that's already mentioned in https://kit.svelte.dev/docs/configuration#alias

```suggestion
	// Path aliases are handled by https://kit.svelte.dev/docs/configuration#alias
```

---

Good point